Bindings/simbody.i

```diff
@@ -175,7 +175,20 @@ namespace SimTK {
 %include <SWIGSimTK/Rotation.h>
 namespace SimTK {
 %template(Rotation) SimTK::Rotation_<double>;
-%template(InverseRotation) SimTK::InverseRotation_<double>;
+
+// TimeSeriesTableMat33 works perfectly as expected, while TimeSeriesTableRotation doesn't
+%template(MatrixBaseMat33)    SimTK::MatrixBase<Mat33>;
+%template(MatrixMat33)        SimTK::Matrix_<Mat33>;
+%template(RowVectorBaseMat33) SimTK::RowVectorBase<Mat33>;
+%template(RowVectorViewMat33) SimTK::RowVectorView_<Mat33>;
+%template(RowVectorMat33)     SimTK::RowVector_<Mat33>;
+
```

Comment:
TimeSeriesTableMat33 now works perfectly and rows can be obtained and used, but doing the same thing for Rotation fails due to the method elementwiseAssign. @aseth1 do you know if this method is supposed to exist for Rotation? To see the error, uncomment the lines below and build

Reply:
Latest version, swig out elementwiseAssign and now the swig generated code compiles and DataTable_<Rotation> acts properly. TimeSeriesTable_<Rotation> however doesn't have the correct class hierarchy likely because it was encountered y swig before both TimeSeriesTable and Rotation were known. Investigating..